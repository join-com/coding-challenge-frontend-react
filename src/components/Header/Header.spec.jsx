import React from "react";
import { shallow } from "enzyme";
import Component from "./Header";

describe("Header component test", () => {
  it("should match snapshot ", () => {
    expect(shallow(<Component />)).toMatchSnapshot();
  });
});
