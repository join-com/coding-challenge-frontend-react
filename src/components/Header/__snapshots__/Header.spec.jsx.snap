// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test should match snapshot  1`] = `
<styled.div>
  <styled.div>
    <Link
      replace={false}
      to="/"
    >
      <Heading
        bold={false}
        level={1}
        uppercase={false}
      >
        Police Department of Berlin
      </Heading>
    </Link>
    <Heading
      bold={false}
      level={3}
      uppercase={false}
    >
      Stolen bikes
    </Heading>
  </styled.div>
</styled.div>
`;
